File: 2018.1/2018-03-07/Adapters/app/src/main/java/br/ufpe/cin/if1001/adapters/ViewHolder.java
package br.ufpe.cin.if1001.adapters;

import android.view.View;
import android.widget.TextView;

class ViewHolder {
    TextView nome = null;
    TextView login = null;

    ViewHolder(View row) {
        this.nome = row.findViewById(R.id.nome);
        this.login = row.findViewById(R.id.login);
    }
}

File: 2019-03-13/Adapters/app/src/main/java/br/ufpe/cin/if710/adapters/ViewHolder.kt
package br.ufpe.cin.if710.adapters

import android.view.View
import android.widget.TextView

internal class ViewHolder(row: View) {
    var nome: TextView? = null
    var login: TextView? = null

    init {
        this.nome = row.findViewById(R.id.nome)
        this.login = row.findViewById(R.id.login)
    }
}